packages/widget_toolkit/lib/src/lib_ui_components/buttons/small_button.dart

```diff
@@ -74,7 +74,7 @@ class SmallButton extends StatelessWidget {
       padding: context.widgetToolkitTheme.smallButtonPadding,
       decoration: BoxDecoration(
         color: state != ButtonStateModel.disabled
-            ? context.widgetToolkitTheme.smallButtonBackgroundColor
+            ? colorStyle?.backgroundColor
```

Comment:
Make sure no breaking changes are introduced. If that's needed the major version of the package needs to be increased

Reply:
I wouldn't consider this a breaking change, I'm just adding the ability to use the color from `colorStyle`. This should break any existing functionality